### PR TITLE
Update mouse navigation

### PR DESCRIPTION
**Pull request by mhkc**
This PR moves the "select region to zoom in" functionality from the overview canvas to the detailed view. It introduces two new input methods `shift - drag mouse` to zoom in on a selected region and `ctrl - mouse click` to zoom out.

**How to test**:
1. Goto [Gens dev version](http://10.0.224.64:5003/643594?region=8:141468087-141468253&variant=84af935d282baeb23862b3c28670f9f9)
2. Try zooming in on the highlighted variant with `shift - drag mouse`
3. Try zooming out with `ctrl - mouse click`
4. Try navigating between chromosomes by clicking them on the overview canvas

**Expected outcome**:
The new input methods is fully working.

**Review:**
- [x] code approved by

### CODE REVIEW

**Comment by ViktorHy** [comment]
works like a clock!

**Review by bjhall — CHANGES_REQUESTED** [review]
The code looks great, I only found some minor bugs.

The biggest one being that jumping directly to a chromosome by pressing for instance 5+Enter seems to be broken. I don't know if this feature is used by anyone? Is it the only reason that the KeyLogger thingy exists?

File: gens/static/js/interactive.js [hunk 1]
```diff
@@ -347,112 +434,91 @@ class InteractiveCanvas {
...
-            this.redraw (this.chromosome + ':0-None');
-            break;
-          case 'a':
-            left(this);
-            break;
-          case 'd':
-            right(this);
-            break;
-          case 'w':
-          case '+':
```
[the remark this answers is not in the document]
Comment (mhkc) [reply gens/static/js/interactive.js:428]: Sounds like a good idea

**Comment by bjhall** [comment]
Looks great! Feel free to merge.

**Pull request merged**